src/Crocodealer/Command/Sync.hs

```diff
@@ -0,0 +1,54 @@
+module Crocodealer.Command.Sync
+       ( sync
+       ) where
+
+import Network.HTTP.Simple (getResponseBody, getResponseStatus, httpBS, parseRequest)
+import Network.HTTP.Types.Status (Status (..))
+
+import Crocodealer.ColorTerminal (errorMessage, infoMessage, warningMessage)
+import Crocodealer.Core.GitHub (FileName (..), RepoName (..), UserName (..))
+
+
+data FileStatus
+    = UpToDate
+    | Outdated
+    | FileDoesNotExist
+
+sync :: UserName -> RepoName -> FileName -> [RepoName] -> IO ()
+sync userName templateRepoName fileName repos =
+    fetchGitHubFile userName templateRepoName fileName >>= \case
+        Nothing -> printFileDiff userName templateRepoName FileDoesNotExist
+        Just f -> do
+            putTextLn $ unFileName fileName
+            forM_ repos $ \repo ->
+                fetchGitHubFile userName repo fileName >>= \case
+                    Nothing -> printFileDiff userName repo FileDoesNotExist
+                    Just r -> do
+                        let fileDiff = calculateDiff f r
+                        printFileDiff userName repo fileDiff
+
+fetchGitHubFile :: UserName -> RepoName -> FileName -> IO (Maybe ByteString)
+fetchGitHubFile userName repoName fileName = do
+    let url = "https://raw.githubusercontent.com/" <> unUserName userName <> "/" <> unRepoName repoName <> "/master/" <> unFileName fileName
+    req <- parseRequest $ toString url
+    resp <- httpBS req
+
+    pure $ case getResponseStatus resp of
+        Status 200 _ -> Just $ getResponseBody resp
+        _            -> Nothing
```

Comment:
Giving it another thought, I wonder, is it really gives us any benefit over just `curl`ing the file content in here? We are mostly not using the http-requests capacity, which gives me thought that we can avoid it in here? What do you think?

Reply:
@vrom911 I agree with this! We indeed can use `curl` via our `shellmet` Haskell library to download the file. It will be even lighter than using all these `http-*` libraries.

Reply:
We can change title of the issue https://github.com/kowainik/crocodealer/issues/21 and use `curl` in a separate PR. I think it worth using `curl` and it will be better than using HTTP libraries, since, as @vrom911 correctly noticed, we need only small subset of what those libraries provide.

Reply:
My concern here would be with requiring the `curl` command to be present on the system this is used on ( unless i'm misunderstanding how the `shellmet` library works ). On Windows for example curl is not present by default if I recall correctly.

Reply:
@Nimor111 we use downloading via `curl` in many of our projects. And `curl` is usually present on systems, even on Windows.

Reply:
@chshersh Alright, then I have no objections.